api/tests/StreamsTest.cpp

```diff
@@ -78,7 +78,6 @@ TEST(StreamsTest, TestGetAPIJSON) {
   auto streams = new Streams(db);
   auto status = Status(StatusCode::OK, "");
   auto valid = streams->Emplace(Channel::Create("test", "youtube", &status));
-  auto removed = streams->Emplace(Channel::Create("jbpratt", "mixer", &status));
```

Comment:
can we switch the service? this test will fail

Reply:
I deleted this

Reply:
But that is incorrect, let me look again :)